Services/JWSProvider/LcobucciJWSProvider.php

```diff
@@ -49,10 +47,10 @@ class LcobucciJWSProvider implements JWSProviderInterface
     /**
      * @throws \InvalidArgumentException If the given crypto engine is not supported
      */
-    public function __construct(KeyLoaderInterface $keyLoader, string $signatureAlgorithm, ?int $ttl, ?int $clockSkew, bool $allowNoExpiration = false, Clock $clock = null)
+    public function __construct(KeyLoaderInterface $keyLoader, string $signatureAlgorithm, ?int $ttl, ?int $clockSkew, bool $allowNoExpiration = false, ClockInterface $clock = null)
```

Comment:
You should also update the service definition to inject the `clock` service defined by FrameworkBundle when it is available (using `on-invalid="null"` to handle cases where the service is disabled) to use the clock configured in the container.